src/dev/riscv/LupV.py

```diff
@@ -175,6 +183,11 @@ def generateDeviceTree(self, state):
         cpus_node.append(FdtPropertyWords("timebase-frequency", [10000000]))
         yield cpus_node
 
+        chosen_node = FdtNode("chosen")
+        chosen_node.append(FdtPropertyStrings("stdout-path",
```

Comment:
I still have been unable to get this to work with "earlycon console=ttyLIO0". As Jason mentioned, if I use kernel parameters "earlycon=lupio_tty,0x20007000" then the earlyconsole does work.